ipfx/dataset/ephys_data_set.py

```diff
@@ -1,10 +1,23 @@
-from typing import Optional, List, Dict, Any, Tuple
-import warnings
+from typing import (
+    Optional, List, Dict, Tuple, Collection, Sequence, Union
```

Comment:
what is the benefit of wrapping imports into parentheses?

Reply:
multiline, slightly prettier than \